internal/crdReplicator/crdReplicator-operator_test.go

```diff
@@ -188,7 +190,7 @@ func TestCRDReplicatorReconciler_UpdateResource(t *testing.T) {
 	networkConfig.SetLabels(map[string]string{"labelTesting": "test"})
 	err = d.UpdateResource(dynClient, gvr, networkConfig, clusterID, consts.OwnershipLocal)
 	assert.Nil(t, err, "error should be nil")
-	obj, err := dynClient.Resource(gvr).Get(context.TODO(), networkConfig.GetName(), metav1.GetOptions{})
+	obj, err := dynClient.Resource(gvr).Namespace("default").Get(context.TODO(), networkConfig.GetName(), metav1.GetOptions{})
```

Comment:
Should be probably replaced with `testNamespace`. The same for all occurrences below.